website/content/en/docs/getting-started/_index.md

```diff
@@ -1,21 +1,82 @@
 ---
 title: Getting Started
-description: Trying ClusterLink for yourself
 weight: 20
 ---
 
+This guide will give you a quick start on installing and setting up the ClusterLink project on a Kubernetes cluster.
+
 ## Prerequisites
 
-<!-- Are there any system requirements for using your project? What languages are supported (if any)? Do users need to already have any software or tools installed? -->
+Before you start, you must have access to a Kubernetes cluster. For example, you can set up a local environment using the [Kind](https://kind.sigs.k8s.io/) project.
 
 ## Installation
 
-<!-- Where can your user find your project code? How can they install it (binaries, installable package, build from source)? Are there multiple options/versions they can install and how should they choose the right one for them? -->
+1. To install ClusterLink on Linux or Mac, use the installation script:
+
+        curl -L <https://Clusterlink.net/install> | sh -
+
+1. Move to the ClusterLink project folder:
+
+        cd clusterlink
+
+1. Export the ClusterLink CLI to your path:
+
+        export PATH=$PWD/bin:$PATH
+
+1. Check the installation by running the command:
+
+        clusterlink version
 
 ## Setup
 
-<!-- Is there any initial setup users need to do after installation to try your project? -->
+To set up ClusterLink on a Kubernetes cluster, follow these steps:
+
+1. Create fabric certificates.
+
+    The ClusterLink Fabric is defined as all Kubernetes clusters (sites) that install ClusterLink gateways and can share services between the clusters, enabling communication among those services.
+    First, create the fabric Certificate Authority(CA):
+
+        clusterlink create fabric --name <fabric_name>
+
+    This command will create the CA files `cert.pem` and `key.pem` in the current folder.
+
+2. Create site certificates.
+
+    Create a site (cluster) certificate:
+
+        clusterlink create site --name <site_name> --fabric <fabric_name>
+
+    This command will create the CA files `cert.pem` and `key.pem` in a  <site_name> folder.
```

Comment:
I don't think a separate folder is needed if you use `<site_name>` and `<fabric_name>` as part of the file names.
Perhaps use `.crt` and `.key` as the suffixes (or `*.crt.pem` if you want to keep the pem indication)?

Reply:
I prefer to key a different folder in case we want to create other files.

Reply:
In that case, I would suggest:
- using `<site_name>` and `<fabric_name>` in the file names
- using `.crt` and `.key` file extensions
- adding `--output <path>` CLI option, defaulting to `<site_name>`